packages/client/src/pages/EmissionsMetricsPage/EmissionsMetricsPage.tsx

```diff
@@ -80,6 +80,15 @@ export default function EmissionsMetricsPage({
                 baseUrl={config.BASE_URL}
                 onApiError={onApiError}
               />
+              {process.env.REACT_APP_ENABLE_REGION_RECOMMENDATION_FEATURE ===
+              'true' ? (
+                <RegionRecommendationCard
+                  data={footprint.filteredData}
+                ></RegionRecommendationCard>
+              ) : null}
+              {process.env.REACT_APP_ENABLE_FORECAST_FEATURE === 'true' ? (
+                <ForecastCard data={footprint.filteredData}></ForecastCard>
+              ) : null}
```

Comment:
`env` variables should be referenced using the `loadConfig` function (and any necessary default values should also be set there).